Explorer/Assets/DCL/PerformanceAndDiagnostics/Analytics/AnalyticsConfiguration.cs

```diff
@@ -102,20 +102,20 @@ private bool TryCreateSegmentConfiguration(out Configuration configuration)
         public void SetWriteKey(string writeKey) =>
             segmentWriteKey = writeKey;
 
-        public bool EventIsEnabled(string eventName)
+        public void Initialize()
         {
             if (eventToggles == null)
-            {
                 eventToggles = new Dictionary<string, AnalyticsEventToggle>();
 
-                foreach (AnalyticsGroup group in groups)
-                foreach (AnalyticsEventToggle eventToggle in group.events)
-                    eventToggles.Add(eventToggle.eventName, eventToggle);
-            }
-
-            return eventToggles.TryGetValue(eventName, out AnalyticsEventToggle toggle) && toggle.isEnabled;
+            foreach (AnalyticsGroup group in groups)
+            foreach (AnalyticsEventToggle eventToggle in group.events)
```

Comment:
Can I get an indent here?

Reply:
It is matter of taste and team agreement on style. I think in this case it reads quite well without indentation, but I will add it for you.